### DISCUSSION TITLE
bro i need help with my twitch chat overlay it locked the settings for the app i delete and re install and the option to edit the chat overlay is still gone

### DISCUSSION THREAD

**Discussion by stunnammg** [baffler/Transparent-Twitch-Chat-Overlay#187] [Q&A]
bro i need help with my twitch chat overlay it locked the settings for the app i delete and re install and the option to edit the chat overlay is still gone

**Comment by baffler** [comment]
You can check out [v0.9.5](https://github.com/baffler/Transparent-Twitch-Chat-Overlay/releases/tag/v0.9.5). You might have to actually install that new version, and uninstall it however, but it will reset your settings after each uninstall now.
